src/core_ops.rs

```diff
@@ -60,7 +60,28 @@ pub fn op_listp(a: &mut Allocator, input: NodePtr, _max_cost: Cost) -> Response
 
 pub fn op_raise(a: &mut Allocator, input: NodePtr, _max_cost: Cost) -> Response {
     let args = Node::new(a, input);
-    args.err("clvm raise")
+    // Quexington requested single argument raise to return the single argument
+    // rather than the full list of arguments.  brun also used to behave this
+    // way... it does make sense to ensure that the full context of the exception
+    // is passed on rather than a more cryptic error in the case of more arguments
+    // though (IMO).  We restrict the single argument case to atoms since it
+    // ensures that a single argument with a pair is distinguishable from multiple
+    // independent arguments (as new, re: richard).
+    let throw_value = args
+        .pair()
+        .as_ref()
+        .and_then(|(_first, _rest)| {
+            _first.atom().map(|_| {
+                if _rest.nullp() {
+                    args.first().unwrap()
+                } else {
+                    args.clone()
```

Comment:
Do we need this clone?

Reply:
i think so.  i can do it a different way.